<launch>
    <node pkg="hrl_feeding_task" type="CupFinder" name="CupFinder" output="screen" respawn="true">
        <remap from="image" to="/head_mount_kinect/rgb/image_color"/>
        <remap from="point_cloud" to="/head_mount_kinect/depth_registered/points"/>
        <remap from="l_mouse_click" to="/clickable_display/l_mouse_click"/>
        <remap from="r_mouse_click" to="/clickable_display/r_mouse_click"/>
        <param name="normal_radius" type="double" value="0.035" />
        <param name="output_frame" value="/torso_lift_link"/>
        <param name="use_closest_pixel" value="True"/>
    </node>

</launch>
